Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/Demo_ScrollPanZoom/Scripts/PanZoom_RectTransf.cs

```diff
@@ -70,6 +70,10 @@ public class PanZoom_RectTransf : PanZoomBase_RectTransf
         [SerializeField]
         private MixedRealityInputAction Zoom_Gesture = MixedRealityInputAction.None;
 
+        [Tooltip("Enable or disable hand gestures for zooming on startup.")]
+        [SerializeField]
+        private bool Zoom_GestureEnabledOnStartup = false;
```

Comment:
the pattern for MRTK has been to use lower case private field names and not using underscores.

Reply:
Good to know . I guess then I have to start a larger cleanup.

Reply:
I just noticed this is in a demo script. while the standards should be the same, I do not believe the team is quite a strict :)

Reply:
Nah, this is a valid ask. We were thinking of moving these over to core at some point as they seem to be useful to the community. I'll update it.

Reply:
Quick follow up question: Entirely lowercase? Or just the beginning of the name lowercase?

Reply:
just the beginning :)